app/index.html

```diff
@@ -24,6 +24,8 @@
   <script type="text/javascript" src="js/livebus/livebus.factory.js"></script>
 
   <script type="text/javascript" src="js/shared/shared.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.directive.js"></script>
```

Comment:
We used to have all similar components in their own folders (i.e. directives would have been in one place) but then we changed according to the Angular style guide (https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#naming) so that related components are in their respective feature folders. I don't think we need to change the CSS background for anything else than the events, so it should be in the events. And if the need arises then we can generalize it. If we generalized all components at first then everything would be in the same place :)